packages/salesforcedx-vscode-lwc/test/vscode-integration/jsconfig.test.ts

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Position, window, workspace, WorkspaceEdit } from 'vscode';
+import URI from 'vscode-uri';
+
+const CONFIG_FILENAME = 'jsconfig.json';
+const TEST_COMPONENT_NAME = 'testComponent';
+const CREATE_COMPONENT_NAME = 'createComponent';
+
+describe('jsconfig Test Suite', () => {
+  let modulesDir: string;
+  let configPath: string;
+  let config: object;
+
+  before(async () => {
+    modulesDir = path.join(
+      workspace.workspaceFolders![0].uri.fsPath,
+      'force-app',
+      'main',
+      'default',
+      'lwc'
+    );
+    configPath = path.join(modulesDir, CONFIG_FILENAME);
+  });
+
+  beforeEach(async function() {
+    this.timeout(5000);
+
+    await createComponent(TEST_COMPONENT_NAME, modulesDir);
+    await waitForConfigUpdate(configPath);
+
+    config = await parseConfig(configPath);
+  });
+
+  it('Should update jsconfig.json on component creation', async function() {
+    this.timeout(5000);
+
+    await createComponent(CREATE_COMPONENT_NAME, modulesDir);
```

Comment:
You don't have to change, but I wanted to share my preference here.
Using constants for these in tests limits my understanding of what the test is doing. 
I have to go hunt for this information, and make assumptions about it otherwise.
So I'm typically okay with the repetition of test data in the test. 

I'm curious about your preferences sometime, we should talk about it.